docs/source/guide/configuration.rst

```diff
@@ -83,6 +83,7 @@ In the following example, a proxy list is set up to use ``proxy.amazon.com``, po
 
     client = boto3.client('kinesis', config=my_config)
 
+Alternatively, you can use the ``HTTP_PROXY`` and ``HTTPS_PROXY`` environment variables to specify proxy servers and the ``NO_PROXY`` environment variable to override proxy servers set by ``HTTP_PROXY`` and  ``HTTPS_PROXY``.  Proxy servers specified using the ``Config`` object will override proxy servers specified using environment variables.
```

Comment:
We may want to split the first sentence into two for readability. We may also want to specifically reference the ``proxies`` Config being what overrides the environment variables. I don't know if @shepazon has any other stylistic comments.